mobile_rewards/src/cli/generate.rs

```diff
@@ -21,7 +20,7 @@ pub struct Cmd {
 impl Cmd {
     pub async fn run(&self) -> Result {
         let store = FileStore::from_env().await?;
-        let mut follower_service = FollowerService::from_env()?;
+        let mut follower_service = FollowerService::from_env().expect("follower_service env error");
```

Comment:
Return the error instead of crashing. That'll mean extending the Error enum

Reply:
How would you recover from this error? Not being able to contact the one thing this service is responsible for talking to feels like a good opportunity to fail fast to indicate to the operator a basic criteria for operation hasn’t been satisfied

Reply:
you don't, and that's why the function returns a `Result` the introduced `expect` ends up crashing the program instead of returning an error

Reply:
[656f9f4](https://github.com/helium/oracles/pull/115/commits/656f9f4d1984816f0f4cb7069a9bed68b81d25c9)